pkg/apis/performanceprofile/v2/performanceprofile_types.go

```diff
@@ -28,7 +28,12 @@ const PerformanceProfilePauseAnnotation = "performance.openshift.io/pause-reconc
 
 // PerformanceProfileEnableRpsAnnotation enables RPS mask setting with systemd for all
 // network devices by including physical interfaces from netdev-rps rule.
-const PerformanceProfileEnableRpsAnnotation = "performance.openshift.io/enable-physical-dev-rps"
+const PerformanceProfileEnablePhysicalRpsAnnotation = "performance.openshift.io/enable-physical-dev-rps"
+
+// PerformanceProfileEnableRpsAnnotation is an emergancy annotation
+// that enables all RPS setting (systemd with udev and crio low latency hook)
+// that overrides workload hint realtime set to false which removes all RPS settings.
+const PerformanceProfileEnableRpsAnnotation = "performance.openshift.io/enable-rps"
```

Comment:
enable-veth-rps? Since we have a physical one too?

Reply:
Hmm I see now, the physical one depends on this one to be set as well. That needs to be documented here however.

Reply:
Also, we are not enabling / disabling RPS here iirc, we are enabling RPS on the container side by this. The RPS mask on host side veth is still supposed to stay, isn't it?

Reply:
this annotation `performance.openshift.io/enable-rps` is meant to remove ALL RPS settings , including host and container side. the use case is when we set realtime workload hint to false but want to ignore it in terms of RPS settings (still set RPS on all sides)
the annotation `performance.openshift.io/enable-physical-dev-rps` is more fine grained and will toggle the udev rule to either contain pci devices or not

Reply:
Ack, so when realtime WH is false, this annotation restores all the veth content + the phys dev restores physical devices on top of that. Is that correct?

The the enables ALL RPS is the confusing part.

Reply:
this annotation restores everything realtime WH = false dropped. meaning it restores: the systemd services created,the crio hook and the crio hook settins = ALL.
the `enable-physical-dev-rps` is placed in a more accurate if directives and will change the creation of udev only (either to contain physical devices or not)
`enable-physical-dev-rps` will not be handled if realtime WH = false AND `enable-physical-dev-rps` = false
for all other cases it will add the physical devices if its true.